pkg/asset/manifests/content/bootkube/kube-system-configmap-etcd-serving-ca.go

```diff
@@ -0,0 +1,23 @@
+package bootkube
+
+import "text/template"
+
+var (
+	// KubeSystemConfigmapEtcdServingCA  is the constant to represent contents of kube-system-configmap-etcd-serving-ca.yaml file
+	KubeSystemConfigmapEtcdServingCA = template.Must(template.New("kube-system-configmap-etcd-serving-ca.yaml").Parse(`
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: etcd-serving-ca
+  namespace: kube-system
+  labels:
+    tier: "control-plane"
+    k8s-app: "kube-apiserver"
+    openshift.io/control-plane: "true"
+    openshift.io/component: "api"
```

Comment:
As [here][1], it would be nice to know what these labels are for.  That makes us less likely to break things accidentally if someone proposes we change/remove one of them later ;).

[1]: https://github.com/openshift/installer/pull/581#discussion_r229774182

Reply:
> As [here](https://github.com/openshift/installer/pull/581#discussion_r229774182), it would be nice to know what these labels are for. That makes us less likely to break things accidentally if someone proposes we change/remove one of them later ;).

@sanchezl they aren't for us.  Remove all labels and the etcd operator can sort out what they want to do.

Reply:
Removed them all.